ocs_ci/ocs/constants.py

```diff
@@ -357,6 +357,43 @@
     PROVIDER_CLIENT_DEPLOYMENT_DIR, "native_storage_client.yaml"
 )
 
+PROVIDER_CLUSTER_RESOURCE_KINDS = [
```

Comment:
Hi @jilju can you pls check if that is legit to pull this into master?

Reply:
This commit is already merged a week ago. Please rebase.

Reply:
@nagendra202 Please check